File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/javascript/cwe-22/code_before2.js
// Source: Row 4 in ./dataset/CVEfixes/Analysis/results/JavaScript/df_javascript_cwe_22.xlsx

var url2FilePath = function(url){
    var subPath = url.substr("/tests/frontend".length);
    if (subPath == ""){
      subPath = "index.html"
    }
    subPath = subPath.split("?")[0];

    var filePath = path.normalize(npm.root + "/../tests/frontend/")
    filePath += subPath.replace("..", "");
    return filePath;
  }

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/cwe-20/ code_before2.c
// Source: Row 10 in ./dataset/CVEfixes/Analysis/results/C/df_c_cwe_20.xlsx

void CIRCNetwork::SetEncoding(const CString& s) {
    m_sEncoding = CZNC::Get().FixupEncoding(s);
    if (GetIRCSock()) {
        GetIRCSock()->SetEncoding(m_sEncoding);
    }
}